kolibri/core/assets/src/vue/textbox/index.vue

```diff
@@ -12,6 +13,7 @@
     :autofocus="autofocus"
     :required="required"
     :type="type"
+    :error="error"
     :invalid="invalid">
   </ui-textbox>
```

Comment:
Need these changes for form feedback, as used in https://github.com/fle-internal/kolibri-instant-schools-plugin/pull/18